coalib/bearlib/languages/definitions/Ruby.py

```diff
@@ -3,4 +3,9 @@
 
 @Language
 class Ruby:
-    pass
+    aliases = 'rb',
+    extensions = '.rb',
+    comment_delimiter = '#'
+    multiline_comment_delimiters = {'=begin': '=end'}
+    string_delimiters = {'"': '"', "'": "'", '%q(': ')', '%Q(': ')', '%(': ')'}
+    encapsulators = {'[': ']', '{': '}', '%w(': ')', '%W(': ')'}
```

Comment:
adding only `%w(...` is misleading.  There is also `%w[...` etc.  Either we list them all, or we need another way to express these.